lms/djangoapps/courseware/tests/test_module_render.py

```diff
@@ -1103,7 +1103,7 @@ def test_static_badlink_rewrite(self):
         result_fragment = module.render(STUDENT_VIEW)
 
         self.assertIn(
-            '/c4x/{org}/{course}/asset/_file.jpg'.format(
+            '/c4x/{org}/{course}/asset/file.jpg'.format(
```

Comment:
Speaking purely as a user, I know that using Studio, you cannot create subdirectories in the assets folder, and cannot upload folders via Files & Uploads.

Import a whole course (that has assets in subdirectories) as a .tar.gz, and you see the assets renamed to "subdir_file.jpg", as mentioned above. A course team that does this needs to make sure their links to assets get updated accordingly. To my best knowledge, only a handful of legacy courses (~12) had to deal with this change.